tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from tests.conftest import params, Helpers
+
+
+logging.basicConfig(level=logging.ERROR)  # external logging level
```

Comment:
I don't see the tests using logging at the moment (unless I'm missing something). We should probably not add this here unless (and until) we plan to add logging to the tests